myst_nbs/gaussian_processes/GP-Latent.myst.md

```diff
@@ -6,50 +6,43 @@ jupytext:
     format_version: 0.13
     jupytext_version: 1.13.7
 kernelspec:
-  display_name: Python (PyMC3 Dev)
+  display_name: pymc
   language: python
-  name: pymc3-dev
+  name: pymc
 ---
 
+```{raw-cell}
+(notebook_name)=
```

Comment:
The target should not be literally `notebook_name` but the actual name of the notebook. The text within the parenthesis is what will be used to reference this notebook from other places in the docs, it needs to be unique to this notebook. https://docs.pymc.io/en/latest/contributing/jupyter_style.html#first-cell

Reply:
Hmm right where it says it in that big red box labeled important that I managed to miss